beacon-chain/rpc/eth/blob/handlers.go

```diff
@@ -131,6 +134,41 @@ func (s *Server) Blobs(w http.ResponseWriter, r *http.Request) {
 		network.WriteError(w, errJson)
 		return
 	}
+	ssz, err := beacon_middleware.SszRequested(r)
+	if err != nil {
+		errJson := &network.DefaultErrorJson{
+			Message: err.Error(),
+			Code:    http.StatusInternalServerError,
+		}
+		network.WriteError(w, errJson)
+		return
+	}
+	if ssz {
+		v2sidecars, err := migration.V1Alpha1BlobSidecarsToV2(sidecars)
+		if err != nil {
+			errJson := &network.DefaultErrorJson{
+				Message: err.Error(),
+				Code:    http.StatusInternalServerError,
+			}
+			network.WriteError(w, errJson)
+			return
+		}
+		sidecarResp := &ethpb.SidecarsResponse{
+			Data: v2sidecars,
+		}
+		sszResp, err := sidecarResp.MarshalSSZ()
+		if err != nil {
+			errJson := &network.DefaultErrorJson{
+				Message: err.Error(),
+				Code:    http.StatusInternalServerError,
+			}
+			network.WriteError(w, errJson)
+			return
+		}
+		network.WriteSsz(w, sszResp, "sidecar_response.ssz")
```

Comment:
The spec defines the return object as "blob sidecars", so `blob_sidecars.ssz` could be more appropriate